test/mason/mason-test/test/docs/exampletest.chpl

```diff
@@ -0,0 +1,12 @@
+// This test is intent for identify if the example from the docs breaks.
```

Comment:
Can you add an empty file called `NOTEST` to this directory to prevent the test runner from trying to run this program as a test?

```
touch  test/mason/mason-test/test/docs/NOTEST
```